testutil/integration/evmos/network/config.go

```diff
@@ -21,20 +21,29 @@ import (
 	evmtypes "github.com/evmos/evmos/v20/x/evm/types"
 )
 
+// defaultChain represents the default EIP155 chain ID used in the suite setup.
+const defaultChain string = utils.MainnetChainID
```

Comment:
nit: the "EIP-155" ID is only the number 9001


```suggestion
// defaultChain represents the default chain ID used in the suite setup.
const defaultChain string = utils.MainnetChainID
```